dco/craig_motlin.dco

```diff
@@ -0,0 +1,10 @@
+1) I, Craig Motlin, certify that all work committed with the commit message
+"covered by: two-sigma-craig_motlin.dco" is copyright
+Two Sigma and that I am authorized by Two Sigma
+to contribute this code under the Apache 2.0 license.
+
+2) I understand and agree all contribution including all personal
+information I submit with it is maintained indefinitely and may be
+redistributed consistent with the open source license(s) involved.
+
+This certification is effective for all code contributed from 2018-03-19 to 2019-03-18.
```

Comment:
This is accurate. I requested permission to contribute back in March when I was working on string operators, even though I never finished that PR.